corehq/apps/app_manager/views.py

```diff
@@ -1002,11 +1002,19 @@ def view_generic(req, domain, app_id=None, module_id=None, form_id=None, is_user
         })
 
         if toggles.FORM_LINK_WORKFLOW.enabled(req.user.username):
+            def qualified_form_name(form):
+                module_name = trans(form.get_module().name, app.langs)
+                form_name = trans(form.name, app.langs)
+                return "{} -> {}".format(module_name, form_name)
+
             modules = filter(lambda m: m.case_type == module.case_type, app.get_modules())
+            if getattr(module, 'root_module_id', None):
+                modules.append(module.root_module)
+            modules.extend(module.get_child_modules())
```

Comment:
Something weird happened and I think this comment got lost, but I was wondering if this would inadvertently add duplicates the module array in the event that a child module has the same case_type as the `module`